src/openforms/forms/tasks.py

```diff
@@ -87,6 +105,33 @@ def recouple_submission_variables_to_form_variables(form_id: int) -> None:
         logger.info("Form variables were updated while this task was runnning.")
 
 
+@app.task(ignore_result=True)
+def recouple_reusable_definition_variables_to_form_variables(form_id: int) -> None:
+    """
+    When the FormVariable bulk create/update endpoint is called, all existing FormVariable related to the form are
+    deleted and new are created. If there are any form definitions which are reusable, we want to update all the forms
+    which are also using these FormDefinitions (concerning the FormVariables). This task updates the FormVariables
+    of each related form in the database, by deleteing the old ones and recreating them (the most safe way).
+    """
```

Comment:
```suggestion
    """
    Fix inconsistencies created by updating a re-usable form definition which is used in other forms.

    When the FormVariable bulk create/update endpoint is called, all existing FormVariable related to the form are
    deleted and new are created. If there are any form definitions which are reusable, we want to update all the forms
    which are also using these FormDefinitions (concerning the FormVariables). This task updates the FormVariables
    of each related form in the database, by replacing the old state with the newly derived set of form variables.
    """
```
in docstrings, you should always provide a one-line summary and the put the extra information below it. See PEP-257 :)